### ISSUE TITLE
Run in background on ios and android

### ISSUE THREAD

**Issue by cagnulein** [cagnulein/qdomyos-zwift#84]
Now everything stops when the app is not in foreground

**Comment by IProgrammedOnce** [comment]
Android
Does this relate to adding QZ to 'unmonitored apps' under 'battery' in settings so QZ is not put to sleep when it loses focus?

**Comment by cagnulein** [comment]
@IProgrammedOnce nope. i mean on android and ios, we need to create a service to let in run completely in background without any garbage collector issue
